docs/whirlpool/docs/03-Whirlpools SDKs/01-Whirlpools/06-Send Transaction.mdx

```diff
@@ -227,12 +117,15 @@ Here's how the transaction message is created:
   </TabItem>
 </Tabs>
 
-### 5: Estimating Compute Unit Limit and Prioritization Fee
-When preparing a transaction, it's important to set a compute unit limit and an appropriate prioritization fee. Setting the compute units too low and the transaction will fail. Setting it too high will make the transaction less favorable for validators to process..
+### 3. Estimating Compute Unit Limit and Prioritization Fee
+Before sending a transaction, it's important to set a compute unit limit and an appropriate prioritization fee.
+
+Transactions that request fewer compute units get high priority for the same amount of prioritization fee (which is defined per compute unit). Setting the compute units too low will result in a failed transaction.
```

Comment:
Nuance: `will result in a failed transaction` -> `might result in your transaction not landing/being included in a block and expiring` or somethuing like that

Reply:
I thought that if you set the compute units too low, you have a _higher_ chance of it being picked up and included, but you risk transaction failure due to compute unit limit.

Reply:
Ahh right yep compute units too low will fail the transaction. My bad read it wrong